docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -131,6 +141,12 @@ is used.
       means that :doc:`/optimizer/cost-based-optimizations` can
       not make smart decisions about the query plan.
     - ``true``
+  * - ``iceberg.projection-pushdown-enabled``
+    - Read only required fields from a struct.
```

Comment:
Would be nice to know from user's perspective what does this mean. I honestly don't understand at the moment the meaning of this setting from this documentation.

Reply:
This is described in the docs. I can add a link here. Will that help ?

Reply:
This property is related to nested data. When dealing with nested data, having this configuration enabled will result in reading only the fields required by the query from the input splits.

https://github.com/trinodb/trino/blob/85c6c09758af6b17d9d8149805125256075ed5ae/plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java#L3482-L3510

cc @alexjo2144

Reply:
please see the current version, I linked docs of projection pushdown